### PR TITLE
chore: walkingpal svg logo

### PR DESCRIPTION
**Pull request by KaranSinghBisht**
#31

### CODE REVIEW

File: assets/svg/logo_svg.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
...AAAIIIICAGHBwEyCAAAIIIBCxgJdKL1c+/zG5796DIQfDjYjvNaojgAACCCDQbQEGHN0WZn0EEEAAAQRSLuDV6ivk3n5cZTdJG3QpLsONLsGyLAIIIIAAAgj8rwADDu4EBBBAAAEEEJAPDr5KSdIecry7C0OO/5H7qWq1vs1jKdxsCCCAAAIIINAtAQYc3ZJlXQQQQAABBAIT8Gr11ZI+Lvd3SVq/Q/F/J+k0NZvfZLjRIVGWQQABBBBAAIEnFGDAwY2BAAIIIIAAAv8n4IODr5X7IXJ/p6T1pkjz+/Hhhtk3bWRk+RTX4nIEEEAAAQQQQOApBRhwcIMggAACCCCAwN8JeLn8epl9XNIuUxhy/EHS6Wo2v26Nxn0QI4AAAggggAAC3RZgwNFtYdZHAAEEEEAgQAEfHHyDkuQQSe+QNGctK/xR0hkaGPiaLV5871pey9cRQAABBBBAAIFJCTDgmBQbFyGAAAIIIJB9AS+X3yjpEJm9XdLsCTa+VtKZkr5qtdo9E7yGryGAAAIIIIAAAlMWYMAxZUIWQAABBBBAILsCXq2+SUlymMx2ljTraZpeJ+ksFQp1Gxq6K7sqNEMAAQQQQACBNAow4EjjqZAJAQQQQACBFAl4tfoWuR8uaSdJ6z5JtOslnSOpZrXanSmKTxQEEEAAAQQQiESAAUckB01NBBBAAAEEJivgkqlabb+L4zBJO8p9nb9by/3PMjtXzeaINRp3THYfrkMAAQQQQAABBKYiwIBjKnpciwACCCCAQCQCvtNOBW266btk1n7x6GslzVxT/a+SzlM+v8SWLLk9Eg5qIoAAAggggEAKBRhwpPBQiIQAAggggEAaBXzXXadrgw12l/vBkl4ls2VyX6hmc8gajdvSmJlMCCCAAAIIIBCPAAOOeM6apggggAACCExZwHfbbR3NmrWH3Esyu0q53CU2PHzrlBdmAQQQQAABBBBAYIoCDDimCMjlCCCAAAIIxCbg8+ZtombzHTK7xkZGfhtbf/oigAACCCCAQDoFGHCk81xIhQACCCCAAAIIIIAAAggggAACayHAgGMtsPgqAggggAACCCCAAAIIIIAAAgikU4ABRzrPhVQIIIAAAggggAACCCCAAAIIILAWAgw41gKLryKAAAIIIIAAAggggAACCCCAQDoF/j8ug0yMecDISwAAAABJRU5ErkJggg=="/>
```
Comment (ashuvssut) [inline assets/svg/logo_svg.svg:7]: Its a PNG. see line `<image id="image0_203_3" width="1080" height="1080" xlink:href="data:image/png;base64,`

> data:image/png

**Comment by ashuvssut** [comment]
You actually need to make this logo using tools like illustrator/figma

you can learn how to create vector graphics. 

let me know if you are going to work further. Else You can close this PR

**Comment by KaranSinghBisht** [comment]
Hey @ashuvssut, I replicated the logo in ai and exported as svg, hope this works!

**Comment by ashuvssut** [comment]
Wow! thanks

**Pull request merged**